### ISSUE TITLE
Drag and Drop event does not get triggered on Windows 11

### ISSUE THREAD

**Issue by WillDeJs** [vizia/vizia#474]
When adding the handler `on_drop` to widgets on windows, the event is not triggered. This was reported on the discord server by a user stating a sample worked on Mac but not on windows. 

This was also reproduced in windows 11 running the following example code.  On dragging a file to the screen and dropping it, the event to print the name of the file to the console does not get triggered. The feature was also tested on Linux on System 76, but behaved properly. See sample code below: 
```rust
use vizia::prelude::*;
fn main() {
    Application::new(move |cx| {
        VStack::new(cx, |cx| {
            HStack::new(cx, |cx| {})
                .height(Pixels(50.0))
                .row_between(Pixels(15.0));
        })
        .on_drop(|cx, dropdata| match dropdata {
            DropData::File(path) => println!("{}", path.to_str().unwrap()),
            DropData::Id(id) => println!("{:?}", id),
        });
    })
    .run().expect("Executing application");
}

```

A potential `winit` bug was initially suspected. However, drag and drop events were tested in the same windows 11 machine with `winit 0.30` and those worked as expected, using the following code:
```rust
use winit::application::ApplicationHandler;
use winit::event::WindowEvent;
use winit::event_loop::{ActiveEventLoop, EventLoop};
use winit::window::{Window, WindowId};

#[derive(Default)]
struct App {
    window: Option<Window>,
}

impl ApplicationHandler for App {
    fn resumed(&mut self, event_loop: &ActiveEventLoop) {
        self.window = Some(
            event_loop
                .create_window(Window::default_attributes())
                .unwrap(),
        );
    }

    fn window_event(&mut self, event_loop: &ActiveEventLoop, _id: WindowId, event: WindowEvent) {
        match event {
            WindowEvent::DroppedFile(path) => println!("We got a file: {}", path.to_str().unwrap()),
            WindowEvent::CloseRequested => event_loop.exit(),
            _ => (),
        }
    }
}
fn main() {
    let event_loop = EventLoop::new().unwrap();
    let mut app = App::default();
    event_loop.run_app(&mut app).unwrap();
}
```

[EDIT]
After opening this issue, I realized Vizia currently uses `winit 0.29` so I came up with an example for that version of the library which behaves as expected on Windows 11. 
```rust
use winit::event::{Event, WindowEvent};
use winit::event_loop::{ControlFlow, EventLoop};
use winit::window::WindowBuilder;

fn main() {
    let event_loop = EventLoop::new().unwrap();
    let _window = WindowBuilder::new().build(&event_loop).unwrap();
    event_loop.set_control_flow(ControlFlow::Poll);
    event_loop
        .run(move |event, elwt| match event {
            Event::WindowEvent {
                event: WindowEvent::DroppedFile(path),
                ..
            } => println!("We got a file: {}", path.to_str().unwrap()),
            Event::WindowEvent {
                event: WindowEvent::CloseRequested,
                ..
            } => elwt.exit(),
            _ => (),
        })
        .unwrap();
}

```

**Comment by Jhonfunk** [comment]
@geom3trik 

I reproduced the issue on windows 11

**Comment by WillDeJs** [comment]
> Possibly fixed by #461

I can confirm, I have just run the same example with the latest Vizia and can confirm it behaved as expected! Thank you so much, Vizia is super cool!
